config/data/india/ihci/canonical_zones.yml

```diff
@@ -0,0 +1,379 @@
+---
```

Comment:
I think after thinking on this a bit I agree with y'all that we should just import all the canonical Regions...it will definitely help us standardize things going forward.